src/pages/search/index.js

```diff
@@ -142,7 +160,30 @@ function SearchPage () {
                                             )
                                         }
                                         <button id="search-more" onClick={() => clickShowMore(index)} >...</button>
-                                        {showMore === index && (<div className="showmore" ref={ref}><Link to={`/profile/${obj.id}`} className="dropprofiles"><div className="dropicon"><ProfileIcon /></div><div className="droptext">Profile</div></Link></div>)}
+                                        {showMore === index && (
+                                        <div>
+
+                                            {cascadeMenu === true && <CascadingMenu />}
```

Comment:
This could be written as just `cascadeMenu` and the next line written as `!cascadeMenu`